app/Http/Controllers/PetController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Pet;
+
+class PetController extends Controller
+{
+    public function show(){
+        return view(
+            'pets',
+            [
+                'pets' => self::getPets(),
+            ]
+        );
+    }
+    
+    public static function getPets(){
```

Comment:
FYI this can be private scope since it's only called within the class definition